tools/pathwaymatcher/pathwaymatcher.xml

```diff
@@ -0,0 +1,381 @@
+                          help="The input follows the Variant Call Format Specification[2] v4.3.
+                          It also allows the possibility to specify only the first 4 columns in the data section of the file:
+                          CHROM, POS, ID, REF.  "/>
+                  </when>
+
+                  <!-- Genes -->
+                  <when value="gene">
+                      <param format="txt" name="input_gene" type="data" label="Genes"
+                          help="File with a one gene name in each line. Genes follow the HUGO gene nomenclature[3]."/>
+                  </when>
+
+                  <!-- Peptides  -->
+                  <when value="peptide">
+                      <param format="txt" name="input_peptide" type="data" label="Simple list"
+                          help="File with a one peptide sequence in each line. You also need to choose a FASTA file to search them against it."/>
+
+                      <param format="fasta" name="input_peptide_database" type="data" label="Protein Database"
+                          help="Select FASTA database from history"/>
+
+                      <param name="ptm_range" type="integer" value="0" label="PTM position range" optional="true"
+                          help="Plus minus positions for the same PTM site."/>
+                  </when>
+
+                  <when value="modifiedpeptide">
+                      <param format="txt" name="input_modifiedpeptide" type="data" label="Peptide List with PTM types and sites"
+                         help="Each line of the file corresponds to a single peptide with post-translational modifications. You also need to choose a FASTA file to search them against it."/>
+
+                      <param format="fasta" name="input_mpeptide_database" type="data" label="Protein Database"
+                          help="Select FASTA database from history"/>
+
+                      <param name="ptm_range" type="integer" value="0" label="PTM position range" optional="true"
+                          help="Plus minus positions for the same PTM site."/>
+                  </when>
+
+                  <!-- Proteins  -->
+                  <when value="uniprot">
+                      <param format="txt" name="input_uniprot" type="data" label="UniProt Accession list"
+                          help="File with a one Uniprot Accession [4] in each line."/>
+                  </when>
+
+                  <when value="ensembl">
+                      <param format="txt" name="input_ensembl" type="data" label="Ensembl identifier list"
+                          help="File with a one Ensembl identifier [5] in each line."/>
+                  </when>
+
+                  <!-- Proteoforms  -->
+                  <when value="proteoforms">
+                      <param format="txt" name="input_proteoforms" type="data" label="Proteoforms"
```

Comment:
this is then a tabular?

Reply:
Well, expected format is a bit of a mix. This is the official documentation about it:
‘’’

Each line must follow any of these formats:

A single protein with no modifications
P00519
A protein with one PTM. The two fields are separated by a ','
P16220;00046:133
A protein and a set of PTMs separated by ';'. The PTMs can be ordered randomly
P62753;00000:235,00000:236,00000:240
In case the PTM type is not known, the modification id used is "00000". For example: "00000:245".


Example:

P00519;00046:245,00048:412
P31749;00047:473,00047:308
P11274;00187:177
P22681;00098:774
P22681
P16220;00046:133
P46109;01192:207
P27361;00047:202,00048:204
Q9UQC2;00000:452
Q15759;00048:182,00047:180
O15530;00048:241
P62753;00048:235,00049:236,00126:240
P12931;00048:419
P40763;00046:705,00046:727
P42229;00048:694

‘’’

Reply:
ok, lets leave it like this for the moment :)